tendrl/integrations/gluster/check_cluster_status.py

```diff
@@ -0,0 +1,23 @@
+import etcd
+
+
+def run():
+    try:
+        clusters = NS._int.client.read("/clusters")
```

Comment:
add a comment that this is only going to be run by the "tendrl/monitor'

Reply:
Sure